lua/packer/luarocks.lua

```diff
@@ -163,11 +163,33 @@ local function run_luarocks(args, disp, operation_name)
   end)
 end
 
+local luarocks_keys = {
```

Comment:
Since this looks like we're only supporting these luarocks options, we should add mention of this to the docs.